src/views/OfferInfoPage/components/OfferDetailSection.tsx

```diff
@@ -1,48 +1,58 @@
 import { styled } from 'styled-components';
 
 import ImgPropLogo from '../assets/images/img_proplogo.png';
+import useGetOfferModel from '../hooks/useGetOfferModel';
 
 import { IcBookmark, IcPin } from '@/views/OfferInfoPage/assets/icons';
 import ConditionContentBox from '@/views/OfferInfoPage/components/ConditionContentBox';
 import { CONDITION_DATA } from '@/views/OfferInfoPage/constants/CONDITION_DATA';
-import { OFFER_DATA } from '@/views/OfferInfoPage/constants/OFFER_DATA';
 
 const OfferDetailSection = () => {
-  const DesingerInfo = OFFER_DATA.data.designerInfo;
-  const OfferDetail = OFFER_DATA.data.offerDetail;
+  const { data } = useGetOfferModel();
+
+  const DesingerInfo = data?.designerInfo;
+  const OfferDetail = data?.styleDetail;
```

Comment:
```suggestion
  const { desingerInfo, offerDetail }  = data?;
```
이렇게 구조분해할당해줘도 되겠죵?? 그리고 변수명은 파스칼케이스가 아니라 카멜케이스로 해줍시다!!